.github/ISSUE_TEMPLATE/release_issue_template.md

```diff
@@ -0,0 +1,40 @@
+# Release tracking issue
+
+<!--
+Note to onduty:
+
+Use comments on this issue to track anything else you think might be of
+interest to the community (e.g. alerting the community to delays).
+
+Once you have filed this issue post a link to it in the #release Slack channel.
+
+As each step is completed, check it off so the community can see the state of
+the current release at a glance.
+
+To document cherry picks that interrupt the default release process, add
+additional checkboxes at the stage the cherry pick was made, e.g. if the
+cherry pick is made after the initial canary release was at 1%, add these lines
+beneath the initial 1% checkbox and then check them as they occur:
+  - [ ] Cherry pick release [<RELEASE_NUMBER>](https://github.com/ampproject/amphtml/releases/tag/<RELEASE_NUMBER>) pushed to Dev Channel
+  - [ ] Cherry pick release pushed to 1%
+
```

Comment:
Add? 
- [ ] Cherry pick release pushed to production (if applicable)

Reply:
Done.